### DISCUSSION TITLE
Drag&Drop Enter / Leave

### DISCUSSION THREAD

**Discussion by sampletext32** [dotnet/Silk.NET#725] [Q&A]
Hi, I am wondering if Silk can dispatch a drag/drop enter and leave events, so I can highlight some UI elements for the end user, when he drags files inside the window and not just the (happened) event, which already exist.

**Comment by HurricanKai** [comment]
I don't believe Silk can do this today. @Perksey ?
Seems like a useful thing we can support in a backwards compatible way.

**Reply by Perksey** [reply]
Currently not, and I don't think we get these events (from both backends at least) today. If this can be done in both GLFW and SDL, happy to add it.
